### PR TITLE
fix: tables

### PR DESCRIPTION
**Pull request by kellyjosephprice**
| [![PR App][icn]][demo] | RM-9793 |
| :--------------------: | :-----: |

## 🧰 Changes

Changes how tables are saved as JSX.

The align attribute is stored on the `Table` component, as well as the table cells:

```
<Table align={["center", "center"]}>
  <thead>
    <tr>
      <th style={{ textAlign: 'center' }}>Hi!</th>
      <th style={{ textAlign: 'center' }}>Hello!</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td style={{ textAlign: 'center' }}>Bye!</td>
      <td style={{ textAlign: 'center' }}>Bye Bye!</td>
    </tr>
  </tbody>
</Table>
```

It also includes some cleanup of code that's not being used, namely `html`, `esast`, and `hastFromHtml`. `hastFromHtml` was going to be used for search indexing, but I think we're going to switch to much simpler parsing of just the `hast`.

## 🧬 QA & Testing

- [Broken on production][prod].
- [Working in this PR app][demo].

[demo]: https://markdown-pr-PR_NUMBER.herokuapp.com
[prod]: https://SUBDOMAIN.readme.io
[icn]: https://user-images.githubusercontent.com/886627/160426047-1bee9488-305a-4145-bb2b-09d8b757d38a.svg

### CODE REVIEW

File: __tests__/transformers/readme-components.test.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ describe('Readme Components Transformer', () => {
     { md: '<Code />', type: 'code' },
     { md: '<CodeTabs />', type: 'code-tabs' },
     { md: '<Image />', type: 'image-block' },
-    { md: '<Table />', type: 'table' },
+    { md: '<Table />', type: 'tableau' },
```
Comment (jennspencer) [inline __tests__/transformers/readme-components.test.ts:9]: 🤌

**Comment by kellyjosephprice** [comment]
Not ready, I keep forgetting.

**Review by trishaprile — APPROVED** [review]
tested locally and looks really good :)

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.75.0-beta.73</b>](/readmeio/markdown/releases/tag/v6.75.0-beta.73)
<br>

**Pull request merged**